portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,58 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Diana's Portfolio</h1>
+      <p>I am STEP intern studying <b>Computer Science</b> in
+         the University of Texas at El Paso, currently 
+         working remotely from Mexico. Here's a picture
+         of one of my dogs, he's name is Remmy! </p><br>
+      <img src = "/myImages/remmy.jpeg" alt = "Remmy" width = "200" height = "300" /> <br>
```

Comment:
Don't hard code styles like height and width. Use css to control it. Example:
`<img src... class="remmy" />`
then in your css do
`.remmy { width: 200px; height: 300px; }`

This is so that you can swap out the css to change the look of the entire page.

Reply:
Sounds good, thanks!